scripts/check_compliance.py

```diff
@@ -333,9 +337,12 @@ def check_no_undef_outside_kconfig(self, kconf):
         # Kconfig files that are not part of the main Kconfig tree, which will
         # trigger false positives until we do something fancier. Skip
         # doc/releases too, which often references removed symbols.
-        grep_cmd = ["git", "grep", "--only-matching", "--line-number", "-I",
-                    "--null", "--perl-regexp",
-                    r"\bCONFIG_[A-Z0-9_]+\b(?!\s*##|[$@{*])",
+
```

Comment:
> // Those would always need to be whitelisted anyway.

While we're here could you explain this (older) comment above? Who are "those"?

Reply:
Piggy-backed a separate commit that cleans up that comment.

I just removed that sentence. Was trying to say that if there was no negative lookahead, then those would always have to be (manually) whitelisted.